dashboard/app/dsl/script_dsl.rb

```diff
@@ -245,39 +234,54 @@ def level(name, properties = {})
       end
     else
       script_level = {
-        lesson: @lesson,
         levels: [level]
       }
 
+      if assessment
+        script_level[:assessment] = assessment
+      end
+      if bonus
+        script_level[:bonus] = bonus
+      end
+      if named
+        script_level[:named_level] = named
+      end
```

Comment:
to match the others and shorten this method, you could turn these each into one-liners:
```
script_level[:assessment] = assessment if assessment
...
```